demo/src/source-info.js

```diff
@@ -12,20 +12,19 @@ const tiffInfo = {
   }
 };
 
+// We store all chunks in the same pyramid so they all have the same URL.
+// TODO: Add logic for when pyramids are in separate files; most similar to tiff.
 const rootZarrUrl =
-  'https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/single_channel_pyramid/img_pyramid/';
+  'https://vitessce-demo-data.storage.googleapis.com/test-data/vanderbilt-data/vanderbilt_mxif_ims.zarr/mxif_pyramid';
 
-const zarrInfo = {
+export const zarrInfo = {
```

Comment:
Should only need the default export?
```suggestion
const zarrInfo = {
```